xbmc/music/Album.cpp

```diff
@@ -176,6 +176,28 @@ std::string CAlbum::GetGenreString() const
   return StringUtils::Join(genre, g_advancedSettings.m_musicItemSeparator);
 }
 
+const std::vector<std::string> CAlbum::GetAlbumArtist() const
+{
+  //Get artist names as vector from artist credits
+  std::vector<std::string> albumartists;
+  for (VECARTISTCREDITS::const_iterator artistCredit = artistCredits.begin(); artistCredit != artistCredits.end(); ++artistCredit)
```

Comment:
Well I'm not sure when the music section started off, but I think we should modernize it where we can. And better to read code is always nice. @Montellese mentioned that format some days ago to me and I think it's new in c++ 11?